packages/components/src/internal/components/picklist/constants.ts

```diff
@@ -1,3 +1,5 @@
 export const PRIVATE_PICKLIST_CATEGORY = 'PrivatePicklist';
 export const PUBLIC_PICKLIST_CATEGORY = 'PublicPicklist';
 export const MAX_SELECTIONS_PER_ADD = 1000;
+export const MAX_SELECTIONS_MESSAGE =
+    'At most ' + MAX_SELECTIONS_PER_ADD.toLocaleString() + ' samples can be selected at once';
```

Comment:
I pulled down this branch and I am not seeing this updated disabled selection message. Am I looking at the wrong button/menu items? Also, it seems like the default message here (the one shown in my screenshot) is sufficient. Was there a need for a custom message for this button?

<img width="535" alt="Screenshot 2024-07-12 at 8 11 37 AM" src="https://github.com/user-attachments/assets/11e8bc21-68f5-4650-91ab-167735253693">

Reply:
The storage item details panel menu options were not being disabled. That's where this is used, but I'll update to be consistent with this message.